server/memphis_helper.go

```diff
@@ -1352,38 +1364,45 @@ func GetMemphisOpts(opts Options, reload bool) (*Account, Options, error) {
 			}
 			accounts = append(accounts, gacc)
 		}
-		// ** not relevant for cloud
-		if reload {
-			appUsers = append(appUsers, &User{
-				Username: "root$1",
-				Password: configuration.ROOT_PASSWORD,
-				Account:  serv.gacc,
-			})
-			appUsers = append(appUsers, &User{
-				Username: MEMPHIS_USERNAME + "$" + strconv.Itoa(1),
-				Password: configuration.CONNECTION_TOKEN,
-				Account:  serv.gacc,
-			})
-			addedTenant[conf.GlobalAccountName] = serv.gacc
-		} else {
-			appUsers = append(appUsers, &User{
-				Username: "root$1",
-				Password: configuration.ROOT_PASSWORD,
-				Account:  gacc,
-			})
-			appUsers = append(appUsers, &User{
-				Username: MEMPHIS_USERNAME + "$" + strconv.Itoa(1),
-				Password: configuration.CONNECTION_TOKEN,
-				Account:  gacc,
-			})
-			addedTenant[conf.GlobalAccountName] = gacc
+		if !CLOUD_ENV {
```

Comment:
Dont call it cloud env, since it won't be relevant also when we will support multi tenancy in the OSS, call it shouldCreateRootUserforGlobalAcc